### PR TITLE
Skeleton spec for UsersController (try #2)

### PR DESCRIPTION
**Pull request by wstephenson**
Looks like I forgot to commit the actual content last night

### CODE REVIEW

File: spec/controllers/users_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe  UsersController do
+  describe 'create' do
+    it 'assigns @user variable with User instance being built from params'
+  end
+
+  describe 'show' do
+     it 'responds successfully with a user object'
+
```
Comment (kalabiyau) [inline spec/controllers/users_controller_spec.rb]: minor style issue

Comment (kalabiyau) [inline spec/controllers/users_controller_spec.rb]: please fix and merge

**Comment by dzyuzin** [comment]
$ rvm all do rspec
fails with
"NonSolus/spec/controllers/users_controller_spec.rb:3:in `<top (required)>': uninitialized constant UsersController (NameError)"
is that ok?

**Comment by wstephenson** [comment]
@dzyuzin Yes, because the class UsersController is coming in #7 but not done yet.

**Pull request merged**